src/collections/_documentation/clients/java/integrations.md

```diff
@@ -1034,3 +1036,31 @@ public ServletContextInitializer sentryServletContextInitializer() {
 ```
 
 After that, your Sentry events should contain information such as HTTP request headers.
+
+#### Spring @Scheduled Tasks
+
+If you want to capture errors from tasks you need also add this configuration:
+
+```java
```

Comment:
This is a Kotlin snippet though:

```suggestion
```kotlin
```

I reckon it would be better to have a Java snippet since it's much more common.
Could you provide a Java version, please? Or if someone has a Java spring app handy to prepare one. Maybe @lobsterkatie does?

Reply:
I do not, but maybe @ndmanvar does?

Reply:
@lobsterkatie -> https://github.com/sentry-demos/java-spring-boot-log4j
let me know if i can be of any help here.